de-lib/src/main/java/org/iplantc/de/analysis/client/views/AnalysisColumnModel.java

```diff
@@ -129,6 +114,10 @@ public String getPath() {
         endDate.setHeader(appearance.endDate());
 
         status.setHeader(appearance.status());
+        status.setMenuDisabled(true);
+        status.setCell(new AnalysisUserSupportCell());
+        status.setSortable(false);
+        status.setHideable(false);
```

Comment:
Also, if we can enable sorting for this column, then we should also enable the menu.
For grids that take a while to load after sorting, the menu is a time-saver when the user knows they want to sort descending in the first place (such as in the data window).
I've encountered columns that are sortable, but the menu is disabled, so I have to sort ascending, wait a second or more for the sort to complete, then click the column header again to get the descending sort I want (and also wait a second or more for the sort to complete in the correct order).